### DISCUSSION TITLE
Welcome to weblogic-monitoring-exporter Discussions!

### DISCUSSION THREAD

**Discussion by russgold** [oracle/weblogic-monitoring-exporter#131] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nxs441** [comment]
Hi, this is a great product! I am wondering about following. Please help to answer:
in values.yml file, WebLogic username and password are hardcoded. Is there a way to encrypt it for security reasons?

Thank you!

**Reply by russgold** [reply]
Thanks for the compliment. `values.yml` is a Prometheus file, and that tool does support use of `htpassword` which would probably do what you want.

https://prometheus.io/docs/guides/basic-auth/

**Comment by dmukherj** [comment]
Can we access custom MBeans with weblogic-monitoring-exporter?

**Reply by russgold** [reply]
I'm afraid not. The only things that you can access are those which are supported by the query feature of the WLS REST API. To do more, it would be necessary to enhance WLS itself.

**Reply by charlesajah** [reply]
what about ServerLifecycleRuntime mbean under domainRuntime? Whenever i add it to the Yaml it still tries to search it under servRuntime hence returns nothing

**Comment by myrulezzz** [comment]
Hi,
I was wondering if you can monitor deployed composites from grafana. Is there an option for wls_exporter to export information and status of deployed composites? (Active, Shutdown, Retired) I am using Soa Infra 12

**Comment by vakhrameev2104** [comment]
HI.
Help me please.  
Tell me about Garbage Collector metrics in WL for exporter? How it works?
I could not found.

**Comment by 162783882** [comment]
Hi.
Install weblogic-exporter v2.1.8 in weblogic domain (12.2.1.3.0), but only Adminserver connect ok With Prometheus.
![prometheus-wls-soa](https://github.com/oracle/weblogic-monitoring-exporter/assets/54291725/2539715f-9064-4077-9f83-6e03e0ab07d8)
ANd I´d like ti know the cause of problem, I only have the error.
Error: server returned HTTP status 500 Internal Server Error
I already intalled it in the capa OSB and Producer layers without problems, but in the SOA layer instances give me this mistake 500.
Please help,  Bye..
